### DISCUSSION TITLE
[Feature] Upload an asset through the share sheet of ios or android.

### DISCUSSION THREAD

**Discussion by akoyaxd** [immich-app/immich#5929] [Feature Request]
### The feature

**Feature discription:** 
Upload an asset through the share sheet of ios or android. So you can quickly choose the immich app as a share target.

**Use case:** 
For people who do not sync every asset on their device or want to store an asset received through an app, that does not store media as a file but keeps it in its appdata, like Signal or Element.

Would be nice to quickly stora an received asset in immich without downloading and uploading it manually in immich.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2659

**Reply by akoyaxd** [reply]
Thanks!
